app/models/job_run.rb

```diff
@@ -97,6 +91,16 @@ def file_manifest
                                   staging_location: batch_context.staging_location)
   end
 
+  def human_state_name
+    return super unless with_preassembly_errors?
+
+    "#{super} (with preassembly errors)"
+  end
+
+  def with_preassembly_errors?
```

Comment:
maybe this should just be a more generic `with_errors?` or `has_errors?` since it seems like it could apply to both discovery report jobs and preassembly jobs?

Reply:
Unfortunately the term preassembly is overloaded in this app. In this case, "preassembly" means errors in discovery report jobs or preassembly jobs (as opposed to an error in accessioning).

Reply:
Added method documentation to make clearer.